engine/include/cubos/engine/fixedstep/plugin.hpp

```diff
@@ -0,0 +1,23 @@
+/// @dir
+/// @brief @ref fixedstep-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup fixedstep-plugin
+
+#pragma once
+
+#include <cubos/engine/prelude.hpp>
+
+namespace cubos::engine
+{
+    /// @defgroup fixedstep-plugin FixedStep
+    /// @ingroup engine
+    /// @brief Adds cubos.fixedStep tag which makes its systems run at a fixed framerate.
+    ///
```

Comment:
```suggestion
    /// @defgroup fixed-step-plugin FixedStep
    /// @ingroup engine
    /// @brief Adds cubos.fixedStep tag which makes its systems run at a fixed framerate.
    ///
```

Add the tags section here, as is done for other plugins.